Tools/Scripts/libraries/webkitapipy/webkitapipy/allow.py

```diff
@@ -91,14 +91,19 @@ def from_dict(cls, doc: dict[str, Any]) -> AllowList:
         seen_clss: dict[Union[str, AllowedSPI.Selector], AllowedSPI] = {}
         for reason in AllowedReason:
             for entry in doc.pop(reason.value, []):
-                syms = entry.pop('symbols', [])
                 clss = entry.pop('classes', [])
                 reqs = entry.pop('requires', [])
                 sels = []
                 for sel in entry.pop('selectors', []):
                     receiver = sel.get('class')
                     sels.append(AllowedSPI.Selector(sel['name'],
                                                     None if receiver == '?' else receiver))
+                # Symbols use C-style namespacing (leading underscore), so that
```

Comment:
In the commit message, you say `C namespace rules, i.e. remove the leading underscore`. Here, `C-style namespacing (leading underscore)` seems to imply the opposite. Is this a typo or am I misunderstanding?

Reply:
It's poorly worded, perhaps I can say "implicit leading underscore"